rt/rs/microprofile-client/src/main/java/org/apache/cxf/microprofile/client/CxfTypeSafeClientBuilder.java

```diff
@@ -76,8 +132,32 @@ public RestClientBuilder executorService(ExecutorService executor) {
                 }
             }
         }
+
+        final String interfaceName = aClass.getName();
+        Optional<Long> timeout = ConfigFacade.getOptionalValue(
+            String.format(REST_CONN_TIMEOUT_FORMAT, interfaceName), Long.class).map(
+                timeoutValue -> {
+                    connectTimeout(timeoutValue, TimeUnit.MILLISECONDS);
+                    return timeoutValue;
+                });
+        if (!timeout.isPresent() && LOG.isLoggable(Level.FINEST)) {
+            LOG.finest("connectionTimeout set by MP Config: " + timeout.get());
```

Comment:
Uh ... `timeout.get()` would blow up since `if` statement has `!timeout.isPresent()` ... Did you mean
```
if (LOG.isLoggable(Level.FINEST)) {
            timeout.ifPresent(t -> 
                LOG.finest("connectionTimeout set by MP Config: " + t)
            );
        };
```?

Reply:
Yes, exactly - good catch!